src/Annotator/ModelAnnotator.php

```diff
@@ -257,8 +261,10 @@ protected function throughAlias(BelongsToMany $association) {
 	 */
 	protected function _junctionTableName(BelongsToMany $association) {
 		$tablesNames = array_map('Cake\Utility\Inflector::underscore', [
-			$association->getSource()->getTable(),
-			$association->getTarget()->getTable()
+			$association->getSource()
+				->getTable(),
+			$association->getTarget()
+				->getTable()
 		]);
 
 		sort($tablesNames);
```

Comment:
These changes are unrelated to this PR (Will be creating a different PR for these). Will fix it up, sorry about that.